test/docs.js

```diff
@@ -189,7 +191,7 @@ describe('documentation', function() {
 
           // in:  fs.anyMethod('--any-file-path--
           // out: fs.anymethod('--a-temporary-file--
-          .replace(/(fs\.[^(]+\(['"])([^'"]*)/g, '$1' + temporaryFile.name);
```

Comment:
Feel free to try anything else you can think of. This module definitely supports Windows, so it's just a matter of figuring out what might be going wrong. The reason I want to use a system temp file is because we inevitably will end up leaving a temp file on disk. If we absolutely can't get this module to work, I'm okay with checking in a "test-file.temp" that's meant just for this use.